recipes/cryohub/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cryohub" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cryohub-{{ version }}.tar.gz
+  sha256: 426ad22409a094bf58765830f85ae19951e7799f76c9286b2b26d08941e63963
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - numpy
+    - pandas
+    - scipy
+    - dask-core
+    - click
+    - pydantic
+    - ipython
+    - cryotypes
+    - dynamotable ==0.2.4
+    - mrcfile ==1.3.0
+    - starfile ==0.4.5
+    - emfile ==0.3
```

Comment:
@brisvag can all these dependencies be relaxed to >= or <= ?

Thanks

Reply:
Pinging @brisvag :)

Reply:
Sorry, yes, they should all be good with `>=`!